jsapp/js/account/usage/usageProjectBreakdown.module.scss

```diff
@@ -1,7 +1,99 @@
 @use 'scss/sizes';
+@use '~kobo-common/src/styles/colors';
 
 .root {
   width: auto;
-  padding-top: sizes.$x40;
+  padding-bottom: sizes.$x40;
+  padding-top: sizes.$x20;
   margin: 0 sizes.$x40;
+  height: 100%;
+  overflow-x: auto;
+}
+
+table {
+  width: 100%;
+  border-collapse: collapse;
+}
+
+.projects {
+  padding-right: 15%;
+  padding-left: 0%;
+  white-space: nowrap;
+}
+
+.wrap {
+  width: sizes.$x120;
+}
+
+th {
+  font-weight: 700;
+  font-size: sizes.$x14;
+  color: colors.$kobo-gray-40;
+  text-align: left;
+  padding: 1.5%;
+}
+
+tr {
+  font-weight: 400;
+  font-size: sizes.$x16;
+  color: colors.$kobo-gray-24;
+}
+
+td {
+  padding: sizes.$x18 sizes.$x8;
+}
+
+td,
+th {
+  border-bottom: sizes.$x1 solid colors.$kobo-gray-92;
+}
+
+.link {
+  color: colors.$kobo-dark-blue;
+  font-weight: 600;
+}
+
+.currentMonth {
+  font-weight: 600;
+}
+
+.pagination {
+  font-weight: 400;
+  font-size: sizes.$x14;
+  color: colors.$kobo-gray-24;
+  margin-top: sizes.$x50;
+  border: sizes.$x1 solid colors.$kobo-gray-85;
+  width: sizes.$x150;
+  border-radius: sizes.$x4;
+  padding: sizes.$x5 0;
+  display: flex;
+  align-items: center;
+  height: sizes.$x36;
+}
+
+button {
+  background: transparent;
+  border: none;
+  padding: 0 sizes.$x8 0 sizes.$x8;
+  color: colors.$kobo-gray-85;
+  cursor: pointer;
+
+  &.active {
+    color: colors.$kobo-gray-24;
+  }
+}
+
+.range {
+  padding: sizes.$x6 sizes.$x6;
+  border-right: sizes.$x1 solid colors.$kobo-gray-85;
+  border-left: sizes.$x1 solid colors.$kobo-gray-85;
+  flex: 1;
+  height: sizes.$x36;
+  width: sizes.$x100;
+  text-align: center;
+  letter-spacing: sizes.$x2;
```

Comment:
This ends up a little scrunched: 
![image](https://github.com/kobotoolbox/kpi/assets/7819986/33c01da4-468c-4ef4-a403-82690d867c0c)
But it un-scrunches if you remove the explicit height, width, and letter-spacing:
![image](https://github.com/kobotoolbox/kpi/assets/7819986/1facfa50-409c-49be-8f8f-d12fab54b8dd)